app/views/users/new.html.erb

```diff
@@ -0,0 +1,42 @@
+<!-- <form accept-charset="UTF-8" method="post" action="/users">
+    <input type="hidden" name="authenticity_token" value="<%= form_authenticity_token %>">
+    <label for="username">Username:-</label>
+    <input type="text" name="user[username]"><br><br>
+    <label for="email">Email:-</label>
+    <input type="email" name="user[email]" /><br><br>
+    <label for="password">Password:-</label>
+    <input type="password" name="user[password]"><br><br>
+    <input type="submit" value="Create">
+</form> -->
+
+<!--<%= form_tag("/users", method: "post") do %>
+<%= label_tag(:username, "Username:") %>
+<%= text_field_tag(:username) %>
+<%= label_tag(:email, "Email:") %>
+<%= email_field_tag(:email) %>
+<%= label_tag(:password, "Password:") %>
+<%= password_field_tag(:password) %>
+<%= submit_tag("Create") %>
+<% end %> -->
+
+<%= form_for @user do |f| %>
+    <%=f.label :username, "username:" %>
+    <%=f.text_field :username%>
+    <%=f.label :email, "email:" %>
+    <%=f.email_field :email%>
+    <%=f.label :password, "password:" %>
+    <%=f.password_field :password%>
+    <%=f.submit "Create" %>
+<%end%>
+
+<% if @post.errors.any? %>
+    <div id="error_explanation">
+      <h2><%= pluralize(@post.errors.count, "error") %> prohibited this post from being saved:</h2>
+
+      <ul>
+      <% @post.errors.full_messages.each do |msg| %>
+        <li><%= msg %></li>
+      <% end %>
+      </ul>
+    </div>
+  <% end %>
```

Comment:
* [ ] There should be absolutely NO logic in views.

  > Views are only for displaying information that became available to it from its corresponding controller not generating it. 

- check this guide * [The Beginner's Guide to Rails Helpers](https://mixandgo.com/learn/the-beginners-guide-to-rails-helpers) that shows how to put if/else statements in a helper method instead of the view [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/section/no_logic_in_helpers_requirement.png) from Pathwright

Reply:
done